addons/binding/org.openhab.binding.chromecast/ESH-INF/thing/thing-types.xml

```diff
@@ -23,9 +51,14 @@
                 <description>Network address of the Chromecast device.</description>
                 <required>true</required>
             </parameter>
+            <parameter name="port" type="integer">
+                <context>network-port</context>
+                <label>Network Port</label>
+                <description>Network port of the Chromecast device.</description>
+                <required>true</required>
```

Comment:
Why not making it optional ? If not set, the API will be called only with the IP adress and a default port will be used.